package.json

```
@@ -26,7 +27,10 @@
     "clean:all": "yarn workspaces foreach -vp run clean && yarn rimraf node_modules",
     "format:all": "yarn prettier --cache --cache-strategy=content --check .",
     "lint:all": "yarn eslint --cache --cache-strategy=content --cache-location=node_modules/.cache/eslint/.eslint-cache .",
-    "test:all": "echo WIP"
+    "build:assisted-ui": "yarn workspace @openshift-assisted/assisted-ui run build",
+    "preview:assisted-ui": "yarn workspace @openshift-assisted/assisted-ui run preview",
+    "test:assisted-ui": "yarn workspace @openshift-assisted/integration-tests-ocm run test",
+    "test:all": "start-server-and-test preview:assisted-ui http://localhost:4173 test:assisted-ui"
```

```suggestion
    "test:all": "yarn workspace @openshift-assisted/integration-tests-ocm run test"
```
Maybe to keep it simple here, we could instead move all the {build,preview}:assisted-ui scripts to the integration-tests-ocm workspace scripts?